# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../vdr/linux-includes.patch
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./dvbdevice.c.orig	2009-04-10 09:54:24.000000000 +0000
+++ ./dvbdevice.c	2009-05-03 15:30:25.000000000 +0000
@@ -7,14 +7,8 @@
  * $Id: dvbdevice.c 2.14 2009/04/10 09:54:24 kls Exp $
  */
 
-#include "dvbdevice.h"
 #include <errno.h>
 #include <limits.h>
-#include <linux/videodev2.h>
-#include <linux/dvb/audio.h>
-#include <linux/dvb/dmx.h>
-#include <linux/dvb/frontend.h>
-#include <linux/dvb/video.h>
 #include <sys/ioctl.h>
 #include <sys/mman.h>
 #include "channels.h"
@@ -27,6 +21,14 @@
 #include "status.h"
 #include "transfer.h"
 
+#include "dvbdevice.h"
+
+#include <linux/videodev2.h>
+#include <linux/dvb/audio.h>
+#include <linux/dvb/dmx.h>
+#include <linux/dvb/frontend.h>
+#include <linux/dvb/video.h>
+
 #define DO_REC_AND_PLAY_ON_PRIMARY_DEVICE 1
 #define DO_MULTIPLE_RECORDINGS 1
 
